lib/index.js

```diff
@@ -218,11 +225,61 @@ class Embark {
     }
 
     if (cmdPlugin) {
-      cmdPlugin.uploadCmds[0].cb();
+      async.waterfall([
+
+        function (callback){
+          engine.logger.debug('building dapp...');
+          engine.startMonitor();
+          engine.startService("libraryManager");
+          engine.startService("web3");
+          engine.startService("pipeline");
+          engine.startService("codeGenerator");
+          engine.startService("deployment");
+          engine.startService("ipfs");
```

Comment:
Since we do that sequence of service starts already at two places in the code, I think we should add a `engine.startAllServices()` function that starts all of those. Unless @iurimatias you think that sometimes we don't want all of them? Though I see that in lib/index.js we do start all the same ones.

Reply:
yeah sometimes we don't want some of them. it's a sort of mix&match depending on the cmd